### DISCUSSION TITLE
Elevated and underground stations

### DISCUSSION THREAD

**Discussion by zhuyuanyue** [voxeltycoon/issues#359] [General]
I think it would be very interesting for trains to travel between buildings and underground。As a Japanese railway fan, l strongly recommend adding more Japanese vehicle models。

**Comment by katepolak** [comment]
Underground stations are planned as part of the subway feature that you can find on the roadmap, as for elevated stations those are mostly just a reskinned variant of the existing stations, so they may already be implemented as a mod, same goes for Japanese vehicles, so I would recommend keeping an eye out for those mods appearing on the workshop.

Marking as "planned" because of the subway part of the suggestion.
